concrete_datastore/concrete/apps.py

```diff
@@ -1,14 +1,19 @@
 # coding: utf-8
+import os
+import glob
 import hashlib
 import logging
+from importlib import import_module
 from collections import defaultdict
 
 from django.utils.translation import ugettext_lazy as _
 from django.apps import AppConfig
+from django.conf import settings
 from django.apps import apps
 
 
 logger_archive_users = logging.getLogger('archive-concrete-users')
+logger_update_pendulum = logging.getLogger('update-pendulum-v2')
```

Comment:
Adding a logger is not necessary, we can merge loggers : for example a system-logger works fine

Reply:
in the existing migrations, we may find an import of `pendulum.pendulum` (pendulum < v2.0). With the v2 of pendulum, django commands crash with `ModuleNotFoundError: No module named 'pendulum.pendulum'`